actix-session/src/storage/redis_rs.rs

```diff
@@ -123,21 +165,33 @@ impl RedisSessionStoreBuilder {
     /// Finalise the builder and return a [`RedisActorSessionStore`] instance.
     ///
     /// [`RedisActorSessionStore`]: crate::storage::RedisActorSessionStore
-    pub async fn build(self) -> Result<RedisSessionStore, anyhow::Error> {
+    #[cfg(feature = "redis-rs-session")]
+    pub async fn build(self) -> Result<RedisSessionStore, Error> {
         let client = ConnectionManager::new(redis::Client::open(self.connection_string)?).await?;
         Ok(RedisSessionStore {
             configuration: self.configuration,
             client,
         })
     }
+
+    /// Finalise the builder and return a [`RedisActorSessionStore`] instance.
+    ///
+    /// [`RedisActorSessionStore`]: crate::storage::RedisActorSessionStore
+    #[cfg(not(feature = "redis-rs-session"))]
+    pub fn build(self) -> RedisSessionStore {
+        RedisSessionStore {
+            configuration: self.configuration,
+            pool: self.pool,
+        }
+    }
 }
 
 impl SessionStore for RedisSessionStore {
     async fn load(&self, session_key: &SessionKey) -> Result<Option<SessionState>, LoadError> {
         let cache_key = (self.configuration.cache_keygen)(session_key.as_ref());
 
         let value: Option<String> = self
-            .execute_command(redis::cmd("GET").arg(&[&cache_key]))
+            .execute_command(cmd("GET").arg(&[&cache_key]))
```

Comment:
IMO these changes are undesirable, reducing readability of the code.

Reply:
I've removed this part because once it's just `redis`, I didn't think there was much need to force it. I'm already reversing that part.